pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -391,6 +391,14 @@ public static class QueryOptionKey {
         public static final String EXPLAIN_PLAN_VERBOSE = "explainPlanVerbose";
         public static final String USE_MULTISTAGE_ENGINE = "useMultistageEngine";
         public static final String ENABLE_NULL_HANDLING = "enableNullHandling";
+        /**
+         * If set, changes the explain behavior in multi-stage engine.
+         *
+         * {@code true} means to ask servers for the physical plan while false means to just use logical plan.
+         *
+         * Use false in order to mimic behavior of Pinot 1.2.0 and previous.
+         */
+        public static final String EXPLAIN_ASKING_SERVERS = "explainAskingServers";
```

Comment:
I think we should update the query option for consistency since we updated the broker config from `pinot.query.explain.ask.servers ` to `pinot.query.multistage.explain.include.segment.plan`. Probably to `explainIncludeSegmentPlan`?